security/subsystem/src/main/java/org/jboss/as/security/remoting/RemotingLoginModule.java

```diff
@@ -123,7 +123,7 @@ public boolean login() throws LoginException {
                         SSLSession session = con.getSslSession();
                         if (session != null) {
                             try {
-                                credential = session.getPeerCertificateChain()[0];
+                                credential = session.getPeerCertificates()[0];
```

Comment:
I think we will probably need a new configuration option - either allow useClientCert to contain another String e.g. new   - or maybe a new option useNewClientCert to switch between the old and new API.